simulation_parameters/microbe_stage/bio_processes.json

```diff
@@ -253,16 +253,16 @@
   "bacterial_thermosynthesis": {
     "Name": "THERMOSYNTHESIS",
     "Inputs": {
-      "temperature": 0.2
+      "temperature": 0.4
```

Comment:
I think this nerfs thermosynthesis harder than I buffed it for 0.8.3... and in 0.8.2 all the feedback we got that it was way too hard to use effectively. So while I'm worried about the other major debuffs to ATP, this seems way too much.

The same applies to radiosynthesis which I did a similar 20% buff (but now this would give a 50% debuff so making it even harder than it was in 0.8.2).

Reply:
I commented on the community forums a but, but I'll put my thoughts here as well. In 0.8.2 nobody commented that thermosynthesis felt like a good gameplay style. For 0.8.3 I have not heard about anything yet from the players about how it feels. So maybe we should try to reach out? But my hunch is that the buff in 0.8.3 was approximately correct and we *should not* debuff the gameplay style at all before we can confirm how it is perceived now.

Ultimately I think that if thermosynthesis or radiosynthesis is too useless *or* too hard for the average player to use, we might as well delete them entirely as they don't provide any gameplay variety as they cannot be relied on as a gameplay style. Thus debuffing them actually majorly reduces the available gameplay variety.